Tests/PsIni.Tests.ps1

```diff
@@ -40,7 +40,7 @@ Describe "PsIni functionality" {
     $dictIn["Category1"]["Key1"] = "Value1"
     $dictIn["Category1"]["Key2"] = "Value2"
     $dictIn["Category2"] = New-Object System.Collections.Specialized.OrderedDictionary([System.StringComparer]::OrdinalIgnoreCase)
-    $dictIn["Category2"]["Key3"] = "Value3"
+    $dictIn["Category2"]["Key3"] = @("Value3.1", "Value3.2")
```

Comment:
One last thing:
please make the sample with 3 entries
this makes clear, that it's a flat array (not `@(@("Value3.1"), "Value3.2")`)`

Reply:
I have added a third value to "key3".

Thank you very much for your support!